docs/keyboard-shortcuts.md

```diff
@@ -0,0 +1,29 @@
+# Keyboard Shortcuts
+
+You can use the following keyboard shortcuts to control the player:
+> Note that those shortcuts will work only if the player is the focused element in the page.
```

Comment:
Just a few items:
1. Change "those" to "these" - everything else is fine.
2. Question @dan-ziv  - do we need to explain what is a focused element? 
3. This looks as though it's written for an end user - if this is for developers, do we need to tell them how to use this? In fact, shouldn't this be included in the Player TV Platform Knowledge Center article (it is, btw)..?